compiler/sigmatch.nim

```diff
@@ -409,7 +409,7 @@ proc handleRange(f, a: PType, min, max: TTypeKind): TTypeRelation =
     elif a.kind == tyRange and
       # Make sure the conversion happens between types w/ same signedness
       (f.kind in {tyInt..tyInt64} and a[0].kind in {tyInt..tyInt64} or
-       f.kind in {tyUInt8..tyUInt32} and a[0].kind in {tyUInt8..tyInt32}) and
+       f.kind in {tyUInt8..tyUInt32} and a[0].kind in {tyUInt8..tyUInt32}) and
```

Comment:
is this a bugfix? if so, it should be in its own PR, also, please provide a test case where it'd fail before and work after

Reply:
Ok, I'll move it. Honestly, I saw the typo and fixed it without giving it too much though, I'll try to find a test case

Reply:
Moved to #17754 with a small test case :)
I'll keep the commit in this PR while the new one is not merged

Reply:
@Menduist you can rebase now that it was merged :)